gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/validation/EndpointGroupsValidationServiceImpl.java

```diff
@@ -81,6 +87,12 @@ public List<EndpointGroup> validateAndSanitize(List<EndpointGroup> endpointGroup
         return endpointGroups;
     }
 
+    private static void validateEndpointConfigInheritance(EndpointGroup endpointGroup, Endpoint endpoint) {
+        if (endpoint.isInheritConfiguration() && isNullOrEmpty(endpointGroup.getSharedConfiguration())) {
```

Comment:
Is it really necessary to force the user to set the value to false? Shouldn't we just ignore the "true" if no config is set at endpoint group level?

Reply:
The inheritConfig flag is false by default, if the flag is set to true to me that mean the user do it with intention. So a null shared configuration is an error.